src/VirtoCommerce.CustomerExportImportModule.Data/Validation/ImportAddressValidator.cs

```diff
@@ -101,46 +107,13 @@ private void AttachValidators()
                             RuleFor(x => x.Record.AddressCountryCode)
                                 .Must((importRecord, countryCode, context) =>
                                 {
-                                    var countries = (IList<Country>) context.ParentContext.RootContextData[Countries];
+                                    var countries = (IList<Country>)context.ParentContext.RootContextData[Countries];
```

Comment:
Now we need to change this validator to support ISO2 too. Right now it will mark all entries with ISO2 as invalid

Reply:
At this moment Code is converted to ISO3 already.